Client/Frontend/Widgets/CurveView.swift

```diff
@@ -0,0 +1,63 @@
+/* Code for drawing the urlbar curve */
```

Comment:
Missing license header. You might want to consider making an Xcode template -- it always puts the Moz header at the top of new Swift files (aligned correctly!) so you don't have to worry about this anymore. I can share mine if you want.